### PR TITLE
implementing: perfect-numbers

### PR DESCRIPTION
**Pull request by Average-user**

### CODE REVIEW

**Comment by Average-user** [comment]
Sorry, I screwed up with git.

File: config.json [hunk 1]
```diff
@@ -69,6 +69,16 @@
       ]
     },
     {
+      "uuid": "7eba3eas-8395-4lu6-ca21-7651s11ab4b4",
```
Comment (petertseng) [inline config.json]: not valid uuid, s, l, and u

Comment (petertseng) [inline config.json]: not valid uuid, s, l, and u

File: config.json [hunk 2]
```diff
@@ -69,6 +69,16 @@
...
+      "slug": "perfect-numbers",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Maybe"
```
Comment (petertseng) [inline config.json]: Number theory, yes, no?

Comment (Average-user) [reply config.json]: Yhea, sorry. I think I had this done before that.

**Pull request merged**